2-mandatory/2-oxygen-levels.js

```diff
@@ -10,8 +10,14 @@
 
     Some string methods that might help you here are .replace() and .substring().
 */
-
-function findSafeOxygenLevel() {}
+function findSafeOxygenLevel(arr) {
+
+  for(let i of arr){
+    if(i.includes('%') && 19.5< i.replace("%","" ) && i.replace("%","" ) < 23.5){
+        return i
```

Comment:
that's nice as you only loop a single time through the array and interrupt as soon as it finds one, but I think the readability could be improved by using some variable name so that it can be easily understood.
What is 19.5? and what is 23.5?
what would happen if I pass this string? "2%0"? The [position](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith) of the % could also be relevant to the data cleaning.